src/Tile/Tile.js

```diff
@@ -34,7 +36,15 @@ const Tile = props => {
         <div
             {...rest}
             className={tileClasses}>
-            {children}
+            {React.Children.toArray(children).map(child => {
+                if (child.type !== TileActions) {
```

Comment:
I think it's slightly safer to do this equality (or non-equality) check using the `displayName` (see `TreeView` code).